aspnetcore/host-and-deploy/blazor/server-side.md

```diff
@@ -5,7 +5,7 @@ description: Learn how to host and deploy a Blazor server-side app using ASP.NET
 monikerRange: '>= aspnetcore-3.0'
 ms.author: riande
 ms.custom: mvc
-ms.date: 06/11/2019
+ms.date: 09/05/2019
 uid: host-and-deploy/blazor/server-side
 ---
 # Host and deploy Blazor server-side
```

Comment:
The Blazor server-side topic seems surprisingly sparse. I think we need to flesh out this topic. Doesn't block this PR though.